handler/users.go

```diff
@@ -13,10 +13,10 @@ import (
 
 type UsersStore interface {
 	List() ([]*gitpods.User, error)
-	GetUser(string) (*gitpods.User, error)
+	GetUser(username string) (*gitpods.User, error)
```

Comment:
To make it more clear that I `GetUser` by username. But maybe instead I should simply call it `GetUserByUsername(string) (*gitpods.User, error)` which I have elsewhere already. :thinking: